### PR TITLE
Integration testing: update readme and quick start

### LINKED ISSUE #1671: Integration testing: quick start missing important notes
Possible errors when following quick start in [test/README](https://github.com/cockpit-project/cockpit/blob/d88b2311ee4d6c5f491974be4514b36fcb834e6e/test/README):
* phantomjs only installed in local cockpit dir, but needed globally for testing
* ``mock`` may issue a warning when user is not in group ``mock``, which leads ``vm-install`` to fail during ``testsuite-prepare``

### PR DESCRIPTION
**Pull request by dperpeet**
fixes #1671

### CODE REVIEW

File: test/README [hunk 1]
```diff
@@ -6,9 +6,11 @@ files for them.
 To run the tests, you need to install the following packages on
 Fedora:
 
-  # yum install trickle nbd-server npm python-libguestfs qemu mock qemu-kvm python \
+  $ yum install trickle nbd-server npm python-libguestfs qemu mock qemu-kvm python \
```
Comment (stefwalter) [inline test/README]: The '#' means execute this as root. If you want to change to '$' then you should probably add sudo to the command.

File: test/README [hunk 2]
```diff
@@ -6,9 +6,11 @@ files for them.
...
          curl libvirt-client libvirtd qemu-nbd fakeroot glibc-static
-  $ npm install phantomjs
+
+Testing requires phantomjs globally on the system, not just inside the cockpit package.
+  $ npm -g install phantomjs
```
Comment (stefwalter) [inline test/README]: Ditto

File: test/README [hunk 3]
```diff
@@ -59,6 +61,8 @@ First, you need to run ./testsuite-prepare.  This will download
 necessary images, compile cockpit, install the result, and set up some
 virtual networking.  It might ask for the sudo password in order to
 setup the network.
```
Comment (stefwalter) [inline test/README]: Missing extra line for paragraph break?

Comment (dperpeet) [inline test/README]: I added a line for the paragraph about mock. The statement `First, you need to run ./testsuite-prepare.` is more of an introduction and shouldn't be separated from the explanation of the process, I think.

**Comment by dperpeet** [comment]
@stefwalter I modified npm to also be called as root, since "-g" flag performs global install

**Pull request closed without merging**